src/components/Header/Header.js

```diff
@@ -3,13 +3,17 @@ import Styled from 'styled-components';
 import NavBar from './NavBar';
 import Logo from './Logo';
 
-
 const StyledHeader = Styled.header`
   display: flex;
   justify-content: space-between;
```

Comment:
One thing that I missed during the last review: `justify-content: space-between;` doesn't really work here as you realized by adding the `margin-left` on the logo. An alternative is to set `flex: 1` on the two text links and `text-algin: right` on the right one. Then those take up equal amount of space while the logo only what's required

But I'll leave it up to you to adjust it